Static/AnalysisLibrary.hs

```diff
@@ -395,7 +400,7 @@ anaLibItem lg opts topLns currLn libenv dg eo itm =
               (alreadyDefined spstr) pos
       else
         -- let (_n, dg''') = addSpecNodeRT dg'' (UnitSig args body) $ show spn
-        return
+        return $ 
```

Comment:
the $ should be removed. Please return to the old code here.

Reply:
done.